admin/class-meta-table-accessible.php

```diff
@@ -8,36 +8,72 @@
  */
 class WPSEO_Meta_Table_Accessible {
 
+	const ACCESSIBLE = '0';
+	const INACCESSBILE = '1';
+
 	/**
 	 * Checks if the given table name exists.
 	 *
 	 * @return bool True when table is accessible.
 	 */
 	public static function is_accessible() {
-		return get_transient( self::transient_name() ) === false;
+		$value = get_transient( self::transient_name() );
```

Comment:
You might consider putting this in its own method: `get_accessible_value` or something like that

Reply:
I'm not convinced of the added value as this functionality is based on the transients and the name is already dynamic.